src/node_crypto.cc

```diff
@@ -877,14 +885,27 @@ void SecureContext::AddRootCerts(const FunctionCallbackInfo<Value>& args) {
         // We do not call back into JS after this line anyway, so ignoring
         // the return value of ProcessEmitWarning does not affect how a
         // possible exception would be propagated.
-        ProcessEmitWarning(sc->env(),
+        ProcessEmitWarning(env,
                            "Ignoring extra certs from `%s`, "
                            "load failed: %s\n",
                            extra_root_certs_file.c_str(),
                            ERR_error_string(err, nullptr));
+      } else {
+        extra_root_certs_file_loaded = true;
```

Comment:
Can we replace this new global boolean with following logic:
```cpp
  if (!extra_root_certs_file.empty()) {
    ...
    if (err) {
      ...
      extra_root_certs_file = nullptr;
    }
  }
...
  static void IsExtraRootCertsFileLoaded(const FunctionCallbackInfo<Value>& args) {
    return args.GetReturnValue().Set(extra_root_certs_file != nullptr);
  }
```

Reply:
The compiler complained about the `extra_root_certs_file != nullptr`. (Otherwise, I think it should be a better choice.)